src/coreComponents/mesh/FaceElementSubRegion.cpp

```diff
@@ -299,7 +324,7 @@ localIndex FaceElementSubRegion::packUpDownMapsImpl( buffer_unit_type * & buffer
 
 localIndex FaceElementSubRegion::unpackUpDownMaps( buffer_unit_type const * & buffer,
                                                    localIndex_array & packList,
-                                                   bool const overwriteUpMaps,
+                                                   bool const,
```

Comment:
`GEOS_UNUSED_PARAM`?

 or if its something we want to obsolete/remove either do so or maybe introduce a new macro to suppress warnings like: `GEOS_TODO_REMOVE_PARAM`.

Reply:
I wrapped this in `GEOS_UNUSED_PARAM`